vars/withPipeline.groovy

```diff
@@ -103,6 +95,18 @@ def call(type, String product, String component, Closure body) {
             product: product,
             component: component)
 
+          if (pipelineConfig.aksStagingDeployment) {
+            sectionCI(
```

Comment:
this seems a bit hacky re-using sectionCI

CI should be for continuous integration not for deployment

Maybe get this PR in and then look at refactoring it?
Not sure

Reply:
Renamed it .